### PR TITLE
Add FIPS support to CLI extension.

### PR DESCRIPTION
**Pull request by JunSun17**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -213,6 +213,9 @@
         - name: --ppg
           type: string
           short-summary: The ID of a PPG.
+        - name: --enable-fips-image
+          type: bool
+          short-summary: Enable FIPS VHD image.
```
Comment (chengliangli0918) [inline src/aks-preview/azext_aks_preview/_help.py:218]: Can we use 'Use FIPS-enabled OS on agent nodes' for short-summary? @palma21

File: src/aks-preview/azext_aks_preview/_help.py [hunk 2]
```diff
@@ -813,6 +816,9 @@
         - name: --os-type
           type: string
           short-summary: The OS Type. Linux or Windows.
+        - name: --enable-fips-image
+          type: bool
+          short-summary: Enable FIPS VHD image.
```
Comment (chengliangli0918) [inline src/aks-preview/azext_aks_preview/_help.py:821]: same comment

File: src/aks-preview/azext_aks_preview/custom.py [hunk 3]
```diff
@@ -1080,6 +1081,7 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
         proximity_placement_group_id=ppg,
         availability_zones=node_zones,
         enable_node_public_ip=enable_node_public_ip,
+        enable_fips=enable_fips_image,
```
Comment (chengliangli0918) [inline src/aks-preview/azext_aks_preview/custom.py:1084]: This change in this file and file blow covers both command 'az aks create...' and 'az aks nodepool add'?

Comment (JunSun17) [reply src/aks-preview/azext_aks_preview/custom.py:1084]: there is another enable_fips=enable_fips_image in this file to cover nodepool add.

File: src/aks-preview/azext_aks_preview/_help.py (hunk 1 above)
Comment (JunSun17) [reply src/aks-preview/azext_aks_preview/_help.py:218]: Sure, changed.

File: src/aks-preview/azext_aks_preview/_help.py (hunk 2 above)
Comment (JunSun17) [reply src/aks-preview/azext_aks_preview/_help.py:821]: Sure, changed.

**Comment by zqingqing1** [comment]
could you add a test?

File: src/aks-preview/azext_aks_preview/_help.py [hunk 4]
```diff
@@ -213,6 +213,9 @@
         - name: --ppg
           type: string
           short-summary: The ID of a PPG.
+        - name: --enable-fips-image
```
Comment (chengliangli0918) [inline src/aks-preview/azext_aks_preview/_help.py:216]: Forgot to ask, does this parameter --enable-fips-image only appear in 'az aks create...' and 'az aks nodepool add...', but not appear in upgrade/scale/update commands?

**Comment by JunSun17** [comment]
> could you add a test?

Test added.

**Review by zqingqing1 — APPROVED** [review]
lgtm

**Pull request merged**